src/node_modules/round-contributors/round-contributors-controller.coffee

```diff
@@ -46,8 +56,18 @@ module.exports = ($scope, $state, config, AuthService, AllocationModel, Allocati
     debug("upload success", response)
     # TODO display success in alerts (see group-members)
     $state.reload('layout')
+    # i want to add an alert after everything is reloaded, have no idea how
+    # $scope.addAlert(
+    #   type: "ya brah",
+    #   msg: "everyone added brah"
+    # )
```

Comment:
currently, after csv successfully uploaded, we use ```$state.reload('layout')``` to reload everything.

i'd like to display a 'success' alert after this reload stuff is done, and I'm not sure how. 

the alert is added and displayed, but the reload rips it away immediately.

any ideas? @ahdinosaur @robguthrie